pkg/cli/cmd/certs.go

```diff
@@ -125,11 +125,13 @@ func (cmd GenerateCertsCmd) Run(c *cc.CommonCtx) error {
 	}
 
 	c.UI.Normal().Msgf("certs directory: %s", certsDir)
-
+	progress := c.UI.Progress("generating dev-certs")
```

Comment:
I had explicitly removed the use of c.UI.Progress why are we bringing this back?